store/tikv/async_commit_test.go

```diff
@@ -395,6 +395,34 @@ func (s *testAsyncCommitSuite) TestAsyncCommitExternalConsistency(c *C) {
 	c.Assert(commitTS2, Less, commitTS1)
 }
 
+func (s *testAsyncCommitSuite) TestAsyncCommitExceedingMaxCommitTS(c *C) {
```

Comment:
Do we need to trigger the failpoint `commitTsTooLarge` here?

Reply:
We needn't. After setting `SafeWindow` and `allowedClockDrift` to zero, it's sure to get a CommitTsTooLarge error.